common/changes/visualTests_2017-04-27-23-02.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Regression Tests",
+      "type": "patch"
```

Comment:
I don't think you need to patch fabric-react for these changes, unless you're changing the product code.